IAMOTZ/ievents
# #152797369 Implement the delete event functionality with react

#### What does this PR do?
It uses react and redux to implement the delete-event functionality consuming the API endpoint for deleting event.
#### Description of Task to be completed?
User should be able to delete an event he/she added
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started, the signup page should be loaded there
- Fill the form and click register
- Add some sample events
- Go to the events page 
- Try deleting an event and it should not appear again
#### Any background context you want to provide?
- The done button on the event cards does something very similar to the delete button but it won't trigger a warning before the events get deleted
#### What are the relevant pivotal tracker stories?
#152797369
#### Screenshots (if appropriate)
![screencapture-localhost-8080-events-1512220628098](https://user-images.githubusercontent.com/26048536/33516041-f3e7d75c-d76b-11e7-91f1-bd2cd9a01c1e.png)
#### Questions:
None
